### PR TITLE
Frontend: Add unit tests for change password & our team controller

### PR DESCRIPTION
**Pull request by Sanji515**
**PR contains the front-end unit tests of:**

- Change Password Controller (`changePwdCtrl.js`)
- Our Team Controller (`ourteamCtrl.js`)

### CODE REVIEW

File: frontend/tests/controllers-test/changePwdCtrl.test.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        it('toggleNewPasswordVisibility', function () {
+            var canShowNewPassword = $rootScope.canShowOldPassword;
+            vm.toggleNewPasswordVisibility();
+            expect($rootScope.canShowNewPassword).toEqual(!canShowNewPassword);
+        });
+
+        it('toggleNewConfirmVisibility', function () {
+            var canShowNewConfirmPassword = $rootScope.canShowOldPassword;
+            vm.toggleNewConfirmVisibility();
+            expect($rootScope.canShowNewConfirmPassword).toEqual(!canShowNewConfirmPassword);
```
Comment (galipremsagar) [inline frontend/tests/controllers-test/changePwdCtrl.test.js]: good ones 👍

**Comment by galipremsagar** [comment]
@Sanji515 , update your branch with changes and validate once.

File: frontend/tests/controllers-test/changePwdCtrl.test.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+describe('Unit tests for Change PWD controller', function () {
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/changePwdCtrl.test.js]: `Unit tests for Change PWD controller` -> `Unit tests for change password controller`

Comment (Sanji515) [reply frontend/tests/controllers-test/changePwdCtrl.test.js]: @RishabhJain2018 done

**Pull request merged**